src/Ganeti/HTools/Node.hs

```diff
@@ -1194,9 +1263,10 @@ showField t field =
     "nmem" -> printf "%5d" $ nMem t
     "xmem" -> printf "%5d" $ xMem t
     "fmem" -> printf "%5d" $ fMem t
+    "umem" -> printf "%5d" $ unallocatedMem t
     "imem" -> printf "%5d" $ iMem t
     "rmem" -> printf "%5d" $ rMem t
-    "amem" -> printf "%5d" $ fMem t - rMem t
+    "amem" -> printf "%5d" $ availMem t
```

Comment:
This is not a equivalent change. The previous one was not bounded by 0. It is probably ok considering that this is just a log, but it still good to check.